.changeset/odd-icons-travel.md

```diff
@@ -0,0 +1,13 @@
+---
+"@ima/plugin-testing-integration": minor
+"@ima/plugin-analytic-fb-pixel": minor
+"@ima/cli-plugin-scramble-css": minor
+"@ima/plugin-analytic-google": minor
+"@ima/cli-plugin-legacy-css": minor
+"@ima/cli-plugin-analyze": minor
+"@ima/plugin-websocket": minor
+"@ima/plugin-analytic": minor
+"@ima/plugin-atoms": minor
+---
+
+dependencies update
```

Comment:
This can be a bit misleading. Only the build dependencies for these packages changed, but not the runtime dependencies.

Reply:
Changeset updated